src/bun.js/webcore/streams.zig

```diff
@@ -118,6 +118,33 @@ pub const ReadableStream = struct {
         ReadableStream__cancel(this.value, globalThis);
     }
 
+    pub fn tee(this: *const ReadableStream, globalThis: *JSGlobalObject) [2]ReadableStream {
+        JSC.markBinding(@src());
+        this.value.unprotect();
+        var branches = ReadableStream__tee(this.value, globalThis);
```

Comment:
Please move this logic to `ReadableStream__tee ` and have `ReadableStream__tee` return the two ReadableStream `JSValue` directly

That will let us skip creating these empty ReadableStream objects (which are expensive to create) and skip the array iterator stuff. You can use `getDirectIndex` on the `JSArray` object

Reply:
OK